scripts/tdexd-run

```diff
@@ -103,4 +185,4 @@ echo ""
 echo "opening market"
 $tdex open
 
-echo '{"market": {"baseAsset": "5ac9f65c0efcc4775e0baec4ec03abdde22473cd3cf33c0419ca290e0751b225","quoteAsset": "'$shitcoin'"}} ' >./test/fixtures/trade.integration.json
+echo '{"market": {"baseAsset": "'$TDEX_BASE_ASSET'","quoteAsset": "'$shitcoin'"}} ' >./test/fixtures/trade.integration.json
```

Comment:
this way you are ovewriting USDT market? the data structure should be an array of markets now?